bool collision
float64 time_elapsed
bool new_run
bool end_of_actino
float64 action_progress
# RLDecisionMsg current_action
bool path_planner_terminate
